packages/venia-ui/lib/components/ProductFullDetail/CustomAttributes/AttributeType/Price/price.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from 'react';
+import { shape, string } from 'prop-types';
+
+import { BrowserPersistence } from '@magento/peregrine/lib/util';
+import { useStyle } from '@magento/venia-ui/lib/classify';
+import { default as PriceComponent } from '@magento/venia-ui/lib/components/Price';
```

Comment:
The default export should already allow declaring your own variable name.
```suggestion
import PriceComponent from '@magento/venia-ui/lib/components/Price';
```